pkg/volume/fc/disk_manager.go

```diff
@@ -57,7 +57,31 @@ func diskSetUp(manager diskManager, b fcDiskMounter, volPath string, mounter mou
 	}
 	err = mounter.Mount(globalPDPath, volPath, "", options)
 	if err != nil {
-		glog.Errorf("failed to bind mount:%s", globalPDPath)
+		glog.V(4).Infof("Mount failed: %v", err)
```

Comment:
I prefer to log error like this instead of log info.
```
glog.Errorf("Failed to bind mount: source:%s, target:%s, err:%v", globalPDPath, volPath, err)
```